From 8ca4824e8b351b34c2b7f2dc327016c79b23ae9d Mon Sep 17 00:00:00 2001
From: Grzegorz Jaszczyk <jaz@semihalf.com>
Date: Mon, 2 Jan 2017 17:43:28 +0100
Subject: [PATCH 1548/2241] arm64: defconfig: enable sysfs interface for GPIOs

After enabling sysfs interface for GPIOs it is possible to configure and
debug the GPIO from the user-space. More information about GPIO's sysfs
interface can be found under Documentation/gpio/sysfs.txt

Change-Id: I1dad97a36cc741122971e66cd56e0ce917d51cac
Signed-off-by: Grzegorz Jaszczyk <jaz@semihalf.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/38530
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Omri Itach <omrii@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
---
 arch/arm64/configs/mvebu_v8_lsp_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/mvebu_v8_lsp_defconfig b/arch/arm64/configs/mvebu_v8_lsp_defconfig
index 9b13bb0..2f964fc 100644
--- a/arch/arm64/configs/mvebu_v8_lsp_defconfig
+++ b/arch/arm64/configs/mvebu_v8_lsp_defconfig
@@ -158,6 +158,7 @@ CONFIG_SPI_ORION=y
 CONFIG_SPI_PL022=y
 CONFIG_SPI_ARMADA_3700=y
 CONFIG_GPIOLIB=y
+CONFIG_GPIO_SYSFS=y
 CONFIG_GPIO_MVEBU_A3700=y
 CONFIG_GPIO_PL061=y
 CONFIG_GPIO_XGENE=y
-- 
2.7.4

